ecs/elections/views.py

```diff
@@ -180,3 +185,52 @@ def form_valid(self, form):
         generator = ElectionGenerator(self.election, **form.cleaned_data)
         generator.generate_elections()
         return super(ElectionGenerateDataFormView, self).form_valid(form)
+
+
+class ScatterChartJSONView(View):
+    def get_data(self, *args, **kwargs):
+        """
+        Returns list of lists of points.
+        First list for candidatates, second for voters.
+        """
+        self.election = Election.objects.get(pk=kwargs['pk'])
+        candidates = self.election.candidates.values_list('position__x', 'position__y')
+        voters = self.election.voters.values_list('position__x', 'position__y')
+        voters_positions = []
+        candidates_positions = []
+        for voter in voters:
+            print voter
+            voters_positions.append(voter)
```

Comment:
A tak w zasadzie to po co przeglądasz listę tylko po to, żeby ją przepisać do innej? :v